packages/genesys-spark-components/src/components/stable/gux-tooltip-title/gux-tooltip-title.tsx

```diff
@@ -132,6 +136,7 @@ export class GuxTooltipTitle {
     this.getTitleElements().forEach(element => {
       if (element.tagName === 'GUX-ICON' && !this.getTitleTextContent()) {
         this.iconOnly = true;
+        this.addIconOnly();
         tooltipTitleText = element.getAttribute('screenreader-text');
       } else if (element.tagName === 'GUX-ICON' && this.getTitleTextContent()) {
         this.addIconDecorative();
```

Comment:
I see this where you got inspiration from but we would not do it this way anymore.